commonDependencyVersionConstraints/build.gradle

```diff
@@ -130,6 +130,7 @@ dependencies {
     implementation group: 'org.apache.tika', name: 'tika-core', version: '1.28.4'
     implementation group: 'com.jayway.jsonpath', name: 'json-path', version: '2.9.0'
     implementation group: 'dnsjava', name: 'dnsjava', version: '3.6.0'
+    api group: 'org.glassfish.jersey.core', name: 'jersey-common', version: '2.34'
```

Comment:
Need to use `api` because a test fixture is part of the dependency tree, for more context see https://github.com/opensearch-project/opensearch-migrations/pull/855#discussion_r1698989107